src/builder.rs

```diff
@@ -30,6 +33,13 @@ impl<W: Write> Builder<W> {
         self.obj.as_mut().unwrap()
     }
 
+    /// Changes the HeaderMode that will be used when reading fs Metadata for
+    /// methods that implicitly read metadata for an input Path. Notably, this
+    /// does _not_ apply to `append(Header)`.
+    pub fn mode(&mut self, mode: HeaderMode) {
+        self.mode = mode;
```

Comment:
Wanted to follow the pattern set with `Entries.raw` here, but was informed that because `Builder` implements `Drop`, that's not possible.